### PR TITLE
patch

### PR DESCRIPTION
**Pull request by pexcn**
- [x] 关闭 boost 的 debug 选项
- [x] 看起来要把 `FORCE_TCP_FASTOPEN` 开了才能支持 TFO?

https://github.com/trojan-gfw/trojan/blob/842ad5bb07eb8bce035fb274571e586629a97c99/src/main.cpp#L97-L106

### CODE REVIEW

File: trojan/Makefile [hunk 1]
```diff
@@ -53,10 +53,10 @@ CMAKE_OPTIONS += \
 	-DENABLE_MYSQL=OFF \
 	-DENABLE_SSL_KEYLOG=ON \
 	-DENABLE_NAT=ON \
-	-DFORCE_TCP_FASTOPEN=OFF \
+	-DFORCE_TCP_FASTOPEN=ON \
```
Comment (wongsyrone) [inline trojan/Makefile:56]: Do NOT try forcing its behavior

File: trojan/Makefile [hunk 2]
```diff
@@ -53,10 +53,10 @@ CMAKE_OPTIONS += \
...
 	-DSYSTEMD_SERVICE=OFF \
 	-DOPENSSL_USE_STATIC_LIBS=TRUE \
-	-DBoost_DEBUG=ON \
+	-DBoost_DEBUG=OFF \
```
Comment (wongsyrone) [inline trojan/Makefile:59]: Just to show more information, doesn't affect the binary.

**Review by wongsyrone — CHANGES_REQUESTED** [review]
NAK.

File: trojan/Makefile (hunk 1 above)
Comment (pexcn) [reply trojan/Makefile:56]: 这个选项在什么情况下才启用呢？
我看到 https://github.com/trojan-gfw/trojan/blob/master/azure-pipelines.yml#L15 这里是开着的

Comment (GreaterFire) [reply trojan/Makefile:56]: This is when you compile trojan on a machine that doesn't support tfo but want tfo to work on another machine.

**Pull request closed without merging**